### DISCUSSION TITLE
allow editing multiple animations at the same time

### DISCUSSION THREAD

**Discussion by CapsAdmin** [CapsAdmin/pac3#413] [Ideas]
**Impulse™'s request**

Hey caps
I have a very very eager request to make, reguarding the animation editor
Can you please. PLEASE. add a dual channel function that adds an extra frame that does not interupt the main frame but plays along with it, and can be edited in the same animation. I think the old animator had a feature like that but it was buggy, and you couldnt animate it. Here's a very low quality visual aid

![](https://cdn.discordapp.com/attachments/273253018793803777/321194934671769601/fuckthisshit.jpg)

I ask this because im trying to animate something VERY FUCKING TEDIOUS and its god damn near impossible to animate the legs moving smoothly with 14 frames. so it comes out as this. 

http://i.imgur.com/dscHmfY.gifv

**Comment by CapsAdmin** [comment]
I removed this feature as it seemed incomplete and not really working at all. I think you can have more than one animation blend together (at least the animation api allows it)

**Comment by thegrb93** [comment]
Someone should make a blender plugin for pac3
